src/components/app-config-ci/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { Typography } from '@equinor/eds-core-react';
+import { debounce } from 'lodash';
+import * as PropTypes from 'prop-types';
+import { useServiceNowApi } from '../../api/use-servicenow-api';
+import { ServiceNowApplicationModel } from '../../models/service-now-application';
+import { useEffect, useRef, useState } from 'react';
+import './style.css';
+import Alert from '../alert';
+import { AxiosError } from 'axios';
+import { MultiValue, SingleValue, StylesConfig } from 'react-select';
+import { ServiceNowApi } from '../../api/service-now-api';
+import { ConfigurationItemPopover } from './ci-popover';
+import { ConfigurationItemSelect } from './ci-select';
```

Comment:
sort import by internal dependencies, external dependencies and stylesheets
```suggestion
import { Typography } from '@equinor/eds-core-react';
import { AxiosError } from 'axios';
import { debounce } from 'lodash';
import * as PropTypes from 'prop-types';
import { useEffect, useRef, useState } from 'react';
import { MultiValue, SingleValue, StylesConfig } from 'react-select';

import { ConfigurationItemPopover } from './ci-popover';
import { ConfigurationItemSelect } from './ci-select';

import { Alert } from '../alert';
import { ServiceNowApi } from '../../api/service-now-api';
import { useServiceNowApi } from '../../api/use-servicenow-api';
import { ServiceNowApplicationModel } from '../../models/service-now-application';

import './style.css';
```